zarr-in-practice.ipynb

```diff
@@ -0,0 +1,3003 @@
+{
```

Comment:
<p>I assume this is intentionally missing because the metadata hasn't been consolidated yet? (I haven't actually worked with Zarr that much yet! 😅) I'd suggest adding one line of text saying that it's not a notebook error that the file doesn't exist yet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>